init.lua

```diff
@@ -423,8 +423,11 @@ function minetest.do_item_eat(hp_change, replace_with_item, itemstack, player, p
 	end
 
 	local itemname = itemstack:get_name()
-	stamina.log("action", "%s eats %s for %s stamina, replace with %s",
-			player:get_player_name(), itemname, hp_change, replace_with_item)
```

Comment:
how about `replace_with_item or "?nil"`?

Reply:
I almost did that originally; however, as "replacing" with "nil" is the norm, i thought it'd make more sense to have a separate log message, to (slightly) reduce the amount of meaningless cruft in the logs.